src/pykeen/datasets/openea.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+
+"""The OpenEA dataset family.
+
+Get a summary with ``python -m pykeen.datasets.openea``
+"""
+
+import logging
+import pathlib
+from abc import ABC
+from typing import Optional, Tuple, cast
+
+import click
+import pandas as pd
+from more_click import verbose_option
+from pystow.utils import download, read_zipfile_csv
+
+from .base import LazyDataset
+from ..triples import TriplesFactory
+from ..typing import LABEL_HEAD, LABEL_RELATION, LABEL_TAIL, TorchRandomHint
+
+__all__ = [
+    "OpenEA",
+]
+
+logger = logging.getLogger(__name__)
+
+GRAPH_PAIRS = ("D_W", "D_Y", "EN_DE", "EN_FR")
+GRAPH_SIZES = ("15K", "100K")
+GRAPH_VERSIONS = ("V1", "V2")
+
+
+class OpenEA(LazyDataset, ABC):
```

Comment:
why ABC?

Reply:
Good question, this is just an artifact from copy-pasting wk3l.py. I'll remove it.